package/kedro_viz/models/flowchart.py

```diff
@@ -580,6 +580,14 @@ def __post_init__(self, data_node: DataNode):
         if not data_node.is_free_input:
             self.run_command = f'kedro run --to-outputs="{data_node.full_name}"'
 
+        # Only check for existence of dataset if we might want to load it.
+        if not (
+            data_node.is_plot_node()
+            or data_node.is_image_node()
+            or data_node.is_tracking_node()
+        ):
+            return
+
```

Comment:
Would it be better to have it as a property or `DataNode`? something like `is_loadable`? At some point there may be other data that we want to load too, for example, a preview pane for a table.

Reply:
Yes, absolutely this would make sense 👍 But I don't want to add a new property until we've figure out how we clean this up in general - I commented more on this in https://github.com/kedro-org/kedro-viz/issues/1043.